src/ui/components/NetworkMonitor/ResponseBody.tsx

```diff
@@ -0,0 +1,54 @@
+import { ResponseBodyData } from "@recordreplay/protocol";
+import React, { useState } from "react";
+import ReactJson from "react-json-view";
+import { TriangleToggle } from "./RequestDetails";
+import { RequestSummary } from "./utils";
+
+const FIVE_MEGABYTES = 5e6;
+
+const safeJSONParseBase64 = (maybeJSON: string) => {
+  try {
+    return [true, JSON.parse(atob(maybeJSON))];
+  } catch (e) {
+    console.log(maybeJSON);
+    console.log(e);
+    return [false, maybeJSON];
+  }
+};
+
+const ResponseBody = ({
```

Comment:
We'll want to share this for request bodies soon so probably better to have a more generic name?